### PR TITLE
Reserve spot for Chicago

### PR DESCRIPTION
**Pull request by angela-li**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,11 @@ We're currently doing up to two **new** events per region per quarter to help en
     + Expected size: <200
     + Quarter: Spring 2019
     + Additional info: 
+- Chicago, USA
```
Comment (daroczig) [inline README.md:125]: can you pls also update the number of slots filled above?

Comment (angela-li) [reply README.md:125]: Yep, I just did!

**Comment by angela-li** [comment]
I updated the number of slots as well, feel free to merge if this looks good to you!

**Comment by daroczig** [comment]
This looks great to me, but I'll let @stephlocke merge

**Comment by stephlocke** [comment]
Welcome to the gang @angela-li - If you keep up with the LA bunch to make sure you don't both run on the same month, that'll help ensure you can get max speaker options! :)

**Comment by angela-li** [comment]
Excellent! I just pinged @amy17519 on Slack. We have lots of folks here in the Chicago R community that would make phenomenal speakers, so I think we shouldn't conflict too much. We're thinking about late April for this event. Very excited for this to happen!

**Comment by amy17519** [comment]
Welcome! @angela-li

**Pull request merged**